libavpipe/src/avpipe_xc.c

```diff
@@ -490,6 +496,21 @@ prepare_decoder(
             elv_dbg("VIDEO STREAM %d, codec_id=%s, stream_id=%d, timebase=%d, xc_type=%d, url=%s",
                 i, avcodec_get_name(decoder_context->codec_parameters[i]->codec_id), decoder_context->stream[i]->id,
                 decoder_context->stream[i]->time_base.den, params ? params->xc_type : xc_none, url);
+            
+            if (decoder_context->video_stream_index == i && decoder_context->format_context->streams[i]->codecpar->width == 0) {
```

Comment:
Do you have an example for this case? How can I generate this? We might be able to solve this in another way...

Reply:
If you remove the code that returns an error here, and run it through the `TestLiveRestarts` qfab integration test a few times, you will get this error. That is why before this PR (in qfab) `EAV_PARAM` was listed as a recoverable error.